src/routing/comm.rs

```diff
@@ -335,9 +342,14 @@ mod tests {
         let mut peer1 = Peer::new().await?;
 
         let message = Bytes::from_static(b"hello world");
-        comm.send(&[peer0.addr, peer1.addr], 1, message.clone())
-            .await
-            .0?;
+        let status = comm
+            .send(&[peer0.addr, peer1.addr], 1, message.clone())
+            .await?;
+
+        assert_matches!(
+            status,
+            SendStatus::AllRecipients | SendStatus::MinDeliveryGroupSizeReached(_)
```

Comment:
expected result shall be just AllRecipients? why `MinDeliveryGroupSizeReached` is also expected?